SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -2,13 +2,12 @@ This document is a declaration of software quality for the `spdlog` external dep
 
 # `spdlog` Quality Declaration
 
-This quality declaration claims that `spdlog` is in the **Quality Level 4** category.
+This quality declaration claims that `spdlog` is in the **Quality Level 1** category.
 
-Though `spdlog` meets many of the criteria stated for software quality, there are key points preventing it from achieving a higher level category.
+Though `spdlog` meets many of the criteria stated for software quality.
 As `spdlog` itself is not a ROS package, many of these gaps could be reconciled in the vendor package that is used to control `spdlog` distribution within ROS.
```

Comment:
I'm not entirely sure what we are trying to say with these two sentences, but it is unclear.  Maybe something like:

```
The upstream code of `spdlog` meets many, but not all of the criteria stated for Quality Level 1.
As `spdlog` is not a ROS package, these gaps are reconciled in the vendor package within ROS 2.
See ./QUALITY_DECLARATION.md for more details.
```